elks/tools/mkromfs.c

```diff
@@ -372,6 +381,17 @@ static int compile_dir (int fd, inode_build_t * inode)
 
 			size += count;
 
+			/* Entry inode index */
+
+			count = write (fd, &entry->inode->index, sizeof (word_t));
+			if (count != sizeof (word_t))
+				{
```

Comment:
I prefer the starting brace for `if while for` etc. to be on the same line. Only functions should have the brace below.

Reply:
OK, I forgot again to apply the indentation rules of this project, sorry. Will not rewrite all the code, but will pay more attention to this point while editing that code.